thriftbp/errors.go

```diff
@@ -16,7 +16,10 @@ type BaseplateErrorCode interface {
 }
 
 var (
-	_ BaseplateErrorCode = (*baseplatethrift.Error)(nil)
+	_                           BaseplateErrorCode = (*baseplatethrift.Error)(nil)
```

Comment:
I don't think this line does anything here. If your intention is to make the `Err*` vars to be of type `BaseplateErrorCode`, this doesn't do that.

Reply:
My formatter just produced that intention after I added lines below.

Reply:
oh I see, that's an already existing line.

I'd suggest add the `Err*` into its own `var` group.